app/models/track.rb

```diff
@@ -40,4 +40,8 @@ def git
   def icon_url
     "https://assets.exercism.io/tracks/ruby-hex-white.png"
   end
+
+  def key_features
+    git.key_features.to_a
+  end
```

Comment:
I could use `delegate:` here if the `to_a` call was omitted. I was unsure if that would be something we would either want to do in the view (I think not) or in the Git model.

Reply:
Yeah. Tradeoffs all round. I think I'd move the `to_a` into the git/track and then delegate here.